server/models/invoice.go

```diff
@@ -8,27 +8,30 @@ import (
 
 type Invoice struct {
 	ID          int              `json:"id" gorm:"primaryKey"`
-	UserID      string           `json:"user_id"  binding:"required"`
+	UserID      string           `json:"user_id" binding:"required"`
 	Total       float64          `json:"total"`
 	Deployments []DeploymentItem `json:"deployments" gorm:"foreignKey:invoice_id"`
 	// TODO:
 	Tax            float64        `json:"tax"`
 	Paid           bool           `json:"paid"`
 	PaymentDetails PaymentDetails `json:"payment_details" gorm:"foreignKey:invoice_id"`
-	LastReminderAt time.Time      `json:"last_remainder_at"`
+	LastReminderAt time.Time      `json:"last_reminder_at"`
 	CreatedAt      time.Time      `json:"created_at"`
 	PaidAt         time.Time      `json:"paid_at"`
+	FileData       []byte         `json:"file_data" gorm:"type:blob"`
```

Comment:
I prefer the option of storing the invoices paths instead of storing the data itself, but if the size of the blob is quite small, then maybe we should go with that